main.py

```diff
@@ -86,7 +88,7 @@ class MechaClient(Client):
 
 
 @Commands.command("ping")
-async def cmd_ping(bot: Client, channel: str, sender: str):
+async def cmd_ping(bot, trigger):
     """
     Pongs a ping. lets see if the bots alive (command decorator testing)
     :param bot: Pydle instance
```

Comment:
Please update docstring, as you replaced parameters without touching the documentation,